docker-compose.yml

```diff
@@ -35,7 +35,7 @@ services:
       start_period: 15s
   
   dev-ticker:
```

Comment:
I guess that locally we dont need the `ZKSYNC_HOME`, since the component in the dev-ticker that use it, is not necessary.

Reply:
Yes, you're right we don't need it, but I preferred to include it to avoid confusion about it in the different envs.
If we want to remove it from here, we could just add a comment in the other `docker-compose.*.yml` files